spec/player_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+
+RSpec.describe Player do
+    it 'has cards and a deck' do
+        card1 = Card.new(:diamond, 'Queen', 12)
+        card2 = Card.new(:spade, '3', 3)
+        card3 = Card.new(:heart, 'Ace', 14)
+        deck = Deck.new([card1, card2, card3])
+
+        expect(card1).to be_an_instance_of(Card)
+        expect(deck).to be_an_instance_of(Deck)
+    end
```

Comment:
This test doesn't have much to do with the `Player` class and would probably be better in the spec files belonging to the class that they are testing.